Yoast/wordpress-seo
# Removed all references to recalibration beta from sidebar and metabox

## Summary

This PR can be summarized in the following changelog entry:

* Removes all references to the recalibration beta from the sidebar and metabox.

## Test instructions

This PR can be tested by following these steps:

* Check that recalibration beta is activated (`SEO -> General -> Features`).
* Open a post.
* Both in the metabox and sidebar, it should say `Readability analysis (beta)` and `Focus keyphrase (beta)`.
* Under `Readability analysis (beta)` and `Focus keyphrase (beta)` there should be a notification with regard to the recalibration beta ("Recalibration beta is active").
* Check out this branch.
* Both in the metabox and sidebar, it should now say `Readability analysis` and `Focus keyphrase`, rather than `Readability analysis (beta)` and `Focus keyphrase (beta)`.
* The notification with regard to the recalibration beta should be gone.

## UI changes

## Quality assurance


Fixes #12264
